remote_attestation/java/tests/com/google/oak/remote_attestation/KeyNegotiatorTest.java

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.oak.remote_attestation;
+
+import com.google.oak.remote_attestation.AeadEncryptor;
+import com.google.oak.remote_attestation.KeyNegotiator;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class KeyNegotiatorTest {
+  private static final byte[] DATA = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+
+  private KeyNegotiator clientKeyNegotiator;
+  private KeyNegotiator serverKeyNegotiator;
+
+  @Before
+  public void setUp() {
+    clientKeyNegotiator = new KeyNegotiator();
+    serverKeyNegotiator = new KeyNegotiator();
+  }
+
+  @Test
+  public void testCreateKeyNegotiator() throws GeneralSecurityException {
+    clientKeyNegotiator.getPublicKey();
```

Comment:
What does this test check? If it is to check that the method does not throw an exception add a comment explaining it. However, something like AssertJ's `doesNotThrowAnyException` is nicer:
```
assertThatCode(() -> {
  // code that should throw an exception
  ...
}).doesNotThrowAnyException();
```

Or perhaps just check that the public key is not empty.

Reply:
Done